jaeger-zipkin/src/main/java/com/uber/jaeger/senders/zipkin/ThriftSpanEncoder.java

```diff
@@ -58,7 +58,7 @@ public Encoding encoding() {
     }
   }
 
-  final ThreadLocal<ReusableTBinaryProtocol> protocol =
+  final static ThreadLocal<ReusableTBinaryProtocol> protocol =
```

Comment:
again, not necessary, and potentially not safe since different instances will be reusing the same protocol